operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -625,14 +617,15 @@ static LongSummaryStatistics summarize(List<Kafka> kafkas, Function<Kafka, Strin
 
     int numReplicasForDefault(long connectionDemand) {
         // use the override if present
+        int desiredReplicaCount = nodeInformer.getList().size() > 0 ? 1:0;
         if (defaultReplicaCount.isPresent()) {
             return defaultReplicaCount.get();
         }
 
         /*
          * an assumption here is that these ingress replicas will not become bandwidth constrained - but that may need further qualification
          */
-        return numReplicasForConnectionDemand(connectionDemand);
+        return Math.max(desiredReplicaCount, numReplicasForConnectionDemand(connectionDemand));
```

Comment:
@rareddy but you want to still bump it up to 2 here, what you previously had should work.

Reply:
oops, misunderstood the previous comment, restored it now, and added a test for it.